website/docs/docs/dbt-versions/dbt-cloud-release-notes-gen.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Weekly dbt release notes"
```

Comment:
Even though this is hidden, it's still possible someone finds it through some path - I think the title and the sidebar should specify these are single tenant release notes